### ISSUE TITLE
Update COSE to latest version 0.9.5

### ISSUE THREAD

**Issue by renzoe** [LudwigSeitz/CWT-Java#2]
Hi! please update COSE maven dependency to use the latest COSE lib
<version>0.9.5</version>
<= 0.9.4 the Encrypt0 structure uses the string "Encrypt1" for the AAD generation, instead of "Encrypt0", making it incompatible. Thank you and thanks for this work! (havent tested yet)

**Comment by LudwigSeitz** [comment]
Fixed. Note that I'd rather not maintain this repo much longer. Have a look at the ACE repo instead.

**Comment by renzoe** [comment]
Hi Ludwig! Thanks yes , understood! Actually I was using an old version of the ACE Repo, and had this same problem, until I found the cause.. then saw the ACE repo it was updated to COSE 0.9.5 some weeks ago.
I am currently not testing the ACE repo, but in the future I will so I will give more feedback to you and the project (I am implementing a Constrained RS , C language. nof fully ACE yet, only basic CBOR capabilities)
